_posts/2020-05-04-strangler-pattern.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: Strangler Pattern
+date: 2020-05-04
+tags: architecture patterns
+author: Tim Myerscough
+image: img/strangler-pattern/Strangler_fig_boulder_katandra.jpg
+---
+
+<center><a href="https://commons.wikimedia.org/wiki/File:Strangler_fig_boulder_katandra.jpg"> <img alt="strangler fig" src="/img/strangler-pattern/Strangler_fig_boulder_katandra.jpg" /></a></center>
+<br/>
+
+Technology is constantly evolving, so thinking in terms of the end goal is unrealistic.  Many companies struggle to keep up with the advances in technology.  If companies lag behind it increases commercial pressure from nimble new entrants, or competitors that can move faster.
+
+Evolutionary architecture, doesn't aim for an end state.  One tool for evolutionary architecture is the [Strangler Pattern](https://martinfowler.com/bliki/StranglerFigApplication.html).
```

Comment:
don't need the comma in the first sentence

Reply:
Is there a good definition for evolutionary architecture you can add here?

Reply:
This whole para feels lost. If you made it about evolutionary architecture it would have some relevance.
Right now it feels like you could delete it and lose nothing.

Reply:
Also I think you're burying the lead.
I think you need to flag the case study here. 
Sure explain the theory but the sexy thing is breaking apart the monolith, taking it to the cloud, severing the gordian knot etc...

Reply:
"Organisations who face replacing legacy applications often face an impossible choice, do you pay the high cost of a big-bang rewrite with the inherent disruption of a cutover, or do you keep the legacy application on life support?

There is a third way." etc